src/control_node_cf.yaml

```diff
@@ -205,6 +205,16 @@ Resources:
 
             su ec2-user -c './control_node_startup_script.sh "${AWS::StackName}" "${Tag}" "${ScriptBucket}"'
 
+  SSHKey:
+    Type: "AWS::SecretsManager::Secret"
+    Properties:
+      Name: !Sub "${KeyName}.pem"
+      SecretString: "not yet uploaded"
```

Comment:
So you are creating here a "dummy" secret that will be updated later when you upload the key?

Reply:
Correct.